test/fixtures.js

```diff
@@ -1862,48 +1862,85 @@ module.exports = function() {
-            "conversation_key": "dummyconversationkey",
-            "contact": {
-                "extra": {},
-                "groups": [],
-                "subscription": {},
-                "msisdn": "+27001",
-                "created_at": "2014-07-28 09:35:26.732",
-                "key": "63ee4fa9-6888-4f0c-065a-939dc2473a99",
+            "url": "http://ndoh-control/api/v1/servicerating/rate/",
+            "data": {
                 "user_account": "4a11907a-4cc4-415a-9011-58251e15e2b4",
-                "name": null,
-                "surname": null,
-                "email_address": null,
-                "dob": null,
-                "twitter_handle": null,
-                "facebook_id": null,
-                "bbm_pin": null,
-                "gtalk_id": null
-            },
-            "answers": {
-                "question_1_friendliness": "very-satisfied",
-                "question_2_waiting_times_feel": "very-satisfied",
-                "question_3_waiting_times_length": "less-than-an-hour",
-                "question_4_cleanliness": "very-satisfied",
-                "question_5_privacy": "very-satisfied"
+                "conversation_key": "dummyconversationkey",
+                "contact": {
+                    "extra": {},
+                    "groups": [],
+                    "subscription": {},
+                    "msisdn": "+27001",
+                    "created_at": "2014-07-28 09:35:26.732",
+                    "key": "63ee4fa9-6888-4f0c-065a-939dc2473a99",
+                    "user_account": "4a11907a-4cc4-415a-9011-58251e15e2b4",
+                    "name": null,
+                    "surname": null,
+                    "email_address": null,
+                    "dob": null,
+                    "twitter_handle": null,
+                    "facebook_id": null,
+                    "bbm_pin": null,
+                    "gtalk_id": null
+                },
+                "answers": {
+                    "question_1_friendliness": "very-satisfied",
+                    "question_2_waiting_times_feel": "very-satisfied",
+                    "question_3_waiting_times_length": "less-than-an-hour",
+                    "question_4_cleanliness": "very-satisfied",
+                    "question_5_privacy": "very-satisfied"
+                }
+            },
+            "response": {
+                "code": 201,
+                "data": {}
+            }
+        }
+    },
+    // Jembi Servicerating Post
+    {
+        "request": {
+            "method": "POST",
+            "headers": {
+                "Authorization": ["Basic " + new Buffer("test:test").toString("base64")],
+                "Content-Type": ["application/json"]
+            },
+            "url": "http://test/v2/json/serviceRating",
+            "data": {
+                "mha": 1,
+                "swt": 1,
+                "msisdn": "+27001",
+                "facility_code": null,
```

Comment:
Service rating can only be done if user has clinic contact extra. Please add it to the test harness user.